### ISSUE TITLE
Add Light Theme for Web UI

### ISSUE THREAD

**Issue by arobinsongit** [netbirdio/dashboard#412]
As an older developer the dark theme (gray on black) can be very difficult for me to read.

Adding the ability to change the web UX to a light theme would be very helpful.  The light theme on the doc page (https://docs.netbird.io/) is perfectly readable and would be a good option.

I have noticed in the top right you have something that appears to be a theme selector button but both Light and System are disabled?  Maybe this is in the works?

**Comment by alindt** [comment]
@arobinsongit Try this image: `docker pull ghcr.io/alindt/netbird-dashboard:feature-light-mode`

**Comment by alindt** [comment]
@lixmal @heisbrot Is this feature not wanted or something?

**Comment by aaronlauterer** [comment]
Another upvote. As a person with astigmatism, I highly prefer dark text on bright backgrounds.
Dark themes usually make the pupils go wider, which reduces the perceived sharpness.

**Comment by kpalang** [comment]
As someone with access to direct sunlight in my working quarters, I, too, need this feature.

**Comment by YouKyi** [comment]
+1

**Comment by LeBaronDeCharlus** [comment]
Would also like this :)

**Comment by IAmNotThisSilverBody** [comment]
I would like to be able to see the dashboard, yeah

**Comment by kamari-nelec** [comment]
please add Light Theme Mode

**Comment by ChillarAnand** [comment]
@heisbrot There are two open PRs to resolve this issue. https://github.com/netbirdio/dashboard/pull/461, https://github.com/netbirdio/dashboard/pull/658

Can you have a look and see how we should go forward to resolve this?

**Comment by ethancls** [comment]
Any news on light mode ? Thanks

**Comment by ChillarAnand** [comment]
This should be done in a week or so.

See https://github.com/netbirdio/dashboard/pull/461#issuecomment-5347708536

**Comment by devaughnsaxa** [comment]
> [@heisbrot](https://github.com/heisbrot) There are two open PRs to resolve this issue. [#461](https://github.com/netbirdio/dashboard/pull/461), [#658](https://github.com/netbirdio/dashboard/pull/658)
> 
> Can you have a look and see how we should go forward to resolve this?

Loving the dedication to project progress. Some people write code, others write reminders.
It's great to see someone so passionate about getting work done by other people.

**Comment by TechHutTV** [comment]
I'll also note this related PR: https://github.com/netbirdio/netbird/pull/7344

**Comment by TechHutTV** [comment]
Light mode in NetBird client is merged and part of RC 0.79. Dashboard is next.

**Comment by ethancls** [comment]
> Light mode in NetBird client is merged and part of RC 0.79. Dashboard is next.

Hello, thanks for your work ! Do you think Dashboard will follow soon ?

**Comment by TechHutTV** [comment]
> Hello, thanks for your work ! Do you think Dashboard will follow soon ?

Yep! Doing the final touches based on feedback from our front end engineer
